### PR TITLE
[terra-menu] Apply theme to menu item

### PR DESCRIPTION
**Pull request by saket2403**
### Summary
Applied theme to menu item to fix clinical theme styles

### Deployment Link

https://terra-framework-deployed-pr-#.herokuapp.com/

### Additional Details

Thank you for contributing to Terra.
@cerner/terra

### CODE REVIEW

File: packages/terra-menu/src/MenuItem.jsx [hunk 1]
```diff
@@ -224,9 +226,13 @@ class MenuItem extends React.Component {
     }
 
     return (
-      <li {...attributes} className={itemClassNames} ref={this.setItemNode} role={role} aria-checked={markAsSelected}>
-        {content}
-      </li>
+      <ThemeContext.Consumer>
```
Comment (emilyrohrbough) [inline packages/terra-menu/src/MenuItem.jsx:229]: I'm not familiar with contextTypes, but is this something that could be provided via the contextTypes like `isGroupItem` and `isSelectableMenu`?

Comment (mjhenkes) [reply packages/terra-menu/src/MenuItem.jsx:229]: You can only read one context at a time with contextTypes. Since it's taken, we need to wrap.

**Comment by mjhenkes** [comment]
So you're applying the theme context here, but you still need to move the theme variables and reference them in the css files right?

**Comment by saket2403** [comment]
> So you're applying the theme context here, but you still need to move the theme variables and reference them in the css files right?

Its there already, I had moved it while applying theme to menu.

**Pull request merged**